packages/eslint-config/package.json

```diff
@@ -33,5 +33,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "resolutions": {
```

Comment:
I'd prefer to bump a version instead of add a resolution if possible. Not sure which one `minimist` is a dependency of though...would have to check.

Reply:
Oh it's eslint

Reply:
It's buried pretty deep and there's not a top-level dependency to upgrade.

Reply:
`yarn upgrade eslint` will update `minimist` (and some other stuff) in the lock file. Looks like [mkdirp fixed it](https://github.com/eslint/eslint/issues/13050#issuecomment-600311496)

Reply:
yeah, I was about to take it back. I think we can just bump the version and fix it.